### ISSUE TITLE
Add SSL support

### ISSUE THREAD

**Issue by duduribeiro** [opensanca/opensanca_jobs#90]
Currently, we only support HTTP. 

Add HTTPS support. Keep in mind that we need to support subdomains too. Like: company1.openjobs.me, company2.openjobs.me.

Probably the best way is to rely on Heroku's Automated Certificate Management https://devcenter.heroku.com/articles/automated-certificate-management

We also should redirect all http requests to https:

https://github.com/rails/rails/blob/b326e82dc012d81e9698cb1f402502af1788c1e9/railties/lib/rails/application/default_middleware_stack.rb#L14

**Comment by samuelgrigolato** [comment]
I've setup Heroku's Certificate Management and submitted #91 to force SSL in production.

Unfortunately this is not enought for [this project](../projects/5), though. HCM does *not* support wildcard certificates yet, as we can see in the section *Known Limitations* of their doc:

> ACM does not support applications that have wildcard custom domains.

So we will need to ditch this in favor of a manual approach. Nevertheless this does not need to be done before the said project is ready for deployment. I think we should postpone this as much as possible so maybe the Heroku guys give us wildcard support in time.

**Comment by duduribeiro** [comment]
Closed by #91
